src/api/render-node.ts

```diff
@@ -162,20 +157,20 @@ function getLogsService(snapshot: StateSnapshot, config: ServiceConfiguration) {
   };
 }
 
-function getChain(vchainId: number, snapshot: StateSnapshot, config: ServiceConfiguration) {
+function getChain(vchainId: number, snapshot: StateSnapshot) {
   const mainVersion = snapshot.CurrentImageVersions['main']['node'];
   if (!mainVersion) return undefined;
   const rolloutGroup = snapshot.CurrentVirtualChains[vchainId.toString()].RolloutGroup;
 
+  const version = snapshot.CurrentImageVersions[rolloutGroup]?.['node'] ?? mainVersion;
   return {
     Id: vchainId,
     InternalPort: 4400,
     ExternalPort: getVirtualChainPort(vchainId),
     InternalHttpPort: 8080,
     Disabled: false,
     DockerConfig: {
-      Image: `${config.DockerNamespace}/node`,
-      Tag: snapshot.CurrentImageVersions[rolloutGroup]?.['node'] ?? mainVersion,
+      ...parseImageTag(version),
```

Comment:
The above parsgeImageTag comments applies to all services (this is the last of them)